apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -175,6 +176,40 @@ export const DEFAULT_BLOCKS: Record<
     title: "Rick Astley - Never Gonna Give You Up",
     url: "https://www.youtube.com/embed/dQw4w9WgXcQ?si=ggGGn4uvFWAIelWD",
   },
+  // TODO: Replace with actual working API endpoint
+  [DYNAMIC_DATA_BANNER_TYPE]: {
+    type: `${DYNAMIC_DATA_BANNER_TYPE}`,
+    apiEndpoint: "https://jsonplaceholder.com/muis_prayers_time",
+    title: "hijriDate",
+    statistics: [
```

Comment:
are we renaming this key to data?

Reply:
addressed in https://github.com/opengovsg/isomer/pull/902/commits/83f4204b5d70bbca484a443a22ff543f03c70eb3